tests/hibernate6/hibernate6-mssql/src/test/resources/application-test.yml

```diff
@@ -2,3 +2,4 @@ test-resources:
   containers:
     mssql:
       accept-license: true
+      image-name: mcr.microsoft.com/mssql/server:2022-latest
```

Comment:
Had to use different image since the default one coming from test resources started failing recently.